src/rf/apps/workers/status_updates.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+
+from datetime import datetime
+
+from apps.core.models import LayerImage, Layer
+import apps.core.enums as enums
+
+
+TIMEOUT_SECONDS = (60 * 10)  # 10 Minutes.
+
+JOB_VALIDATE = [
+    'ObjectCreated:CompleteMultipartUpload',
+    'ObjectCreated:Put',
+    'ObjectCreated:Post'
+]
+JOB_HANDOFF = 'emr_handoff'
+JOB_TIMEOUT = 'timeout'
+
+TIMEOUT_DELAY = 60
+
+
+def mark_image_valid(s3_uuid):
+    try:
+        image = LayerImage.objects.get(s3_uuid=s3_uuid)
+        image.status = enums.STATUS_VALID
+        image.save()
+        return True
+    except (ValueError, LayerImage.DoesNotExist):
+        return False
+
+
+def mark_image_invalid(s3_uuid, error_message):
+    try:
+        image = LayerImage.objects.get(s3_uuid=s3_uuid)
```

Comment:
Regarding UUID field... I ran into this problem earlier that I thought worth mentioning. We use a consistent naming convention throughout the code but it's a little misleading. 

Ex.

```python
In [15]: s3_uuid = '68fd4a89-de8c-4644-8036-59047775f546'

In [16]: model = LayerImage.objects.get(s3_uuid=s3_uuid)

In [17]: model.s3_uuid == s3_uuid
Out[17]: False
```

This makes sense but can be confusing depending on the context.